zerver/tests/test_new_users.py

```diff
@@ -26,7 +27,8 @@ def test_send_login_emails_if_send_login_email_setting_is_true(self):
 
```

Comment:
Do we want to pass a `User-Agent` for at least one of these so we have a test with a non-unknown value?  I think to do that we'd need to not use `self.login` at all and actually hit the `/login` endpoint (I think there's some example code that does that in `test_signup.py`; you can find it via grep I imagine)